src/core_modules/capture-core/components/CardList/CardListItem.component.js

```diff
@@ -10,14 +10,14 @@ import type { CardDataElementsInformation, SearchResultItem } from '../Pages/Sea
 import type { DataElement } from '../../metaData';
 import { availableCardListButtonState, enrollmentTypes } from './CardList.constants';
 import { ListEntry } from './ListEntry.component';
+import { dataElementTypes } from '../../metaData';
 
 type OwnProps = $ReadOnly<{|
     item: SearchResultItem,
     currentSearchScopeName?: string,
-    currentProgramId?: string,
-    getCustomTopElements?: ?(props: Object) => Element<any>,
-    getCustomBottomElements?: ?(props: Object) => Element<any>,
-    imageDataElement: DataElement,
+    currentProgramId: ?string,
```

Comment:
Now that the Card is displayed in a uniformed way around the app. `currentProgramId` is not an optional property. 
It is used to display information such as `orgUnitName` and `enrollmentDate`.